Research/Research.xcodeproj/project.pbxproj

```diff
@@ -1078,6 +1078,10 @@
 		FF8B54EE1FCE6D15006B6937 /* RSDExceptionHandler.m in Sources */ = {isa = PBXBuildFile; fileRef = FFF159DF1FB4D7A60061BA93 /* RSDExceptionHandler.m */; };
 		FF9557CC23060F4B00B8D60B /* RSDAppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = FF9557CB23060F4B00B8D60B /* RSDAppDelegate.swift */; };
 		FF9C264B243FA48B00241202 /* QuestionTableDataSourceTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = FF9C264A243FA48B00241202 /* QuestionTableDataSourceTests.swift */; };
+		FF9F343724EB473600736F04 /* LocalizationBundle.swift in Sources */ = {isa = PBXBuildFile; fileRef = FF9F343624EB473600736F04 /* LocalizationBundle.swift */; };
```

Comment:
looks like somehow this got added 4x to the project file...

Reply:
Yes -> macOS, tvOS, iOS, watchOS